regression/scripts/issue_01161.ys

```diff
@@ -0,0 +1,13 @@
+read_verilog ../top.v
+
+hierarchy
+proc
+
+write_ilang error_design.proc.il
+show -format svg -prefix error_design.proc error_design
```

Comment:
I don't think `show` is necessary to exhibit this bug?

Reply:
You are right. Didn't clean it after copying from issue description.

Reply:
fixed